src/styles/blocks/page.scss

```diff
@@ -0,0 +1,23 @@
+.page {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+
+  &__header {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 50px;
+    align-items: center;
+  }
+
+  &__main {
+    display: grid;
+    grid-template-columns: repeat(1, 200px);
+    justify-content: center;
+    gap: 46px 48px;
+    padding: 50px 40px;
+
+    @include set-columns(488px, 2);
+    @include set-columns(768px, 3);
+    @include set-columns(1024px, 4);
```

Comment:
Ensure that the mixin 'set-columns' is defined in your project. If it's not, this will lead to an error. If it is defined, make sure it is correctly implemented to handle the parameters being passed.